### PR TITLE
Pqts 21/allow set an avatar only for superusers and higher

### PR DESCRIPTION
**Pull request by yuliatokaryk**

### CODE REVIEW

File: app/models/profile.rb [hunk 1]
```diff
@@ -7,4 +8,12 @@ class Profile < ApplicationRecord
   def full_name
     "#{first_name} #{last_name}"
   end
+
+  private
+
+  def remove_avatar
+    return unless user.role == 'user'
+
+    self.avatar = nil
```
Comment (ihortok) [inline app/models/profile.rb]:
```suggestion
    avatar.purge
```

File: app/models/profile.rb [hunk 2]
```diff
@@ -7,4 +8,12 @@ class Profile < ApplicationRecord
   def full_name
     "#{first_name} #{last_name}"
   end
+
+  private
+
+  def remove_avatar
```
Comment (ihortok) [inline app/models/profile.rb]:
```suggestion
  def check_if_user_is_allowed_to_have_avatar
```

**Review by ihortok — CHANGES_REQUESTED** [review]

File: app/models/profile.rb (hunk 2 above)
Comment (yuliatokaryk) [reply app/models/profile.rb]: [bb13af1](https://github.com/yuliatokaryk/quotes/pull/50/commits/bb13af1b961761bfcf66354aa5f84b8cc26f1764)

File: app/models/profile.rb (hunk 1 above)
Comment (yuliatokaryk) [reply app/models/profile.rb]: [bb13af1](https://github.com/yuliatokaryk/quotes/pull/50/commits/bb13af1b961761bfcf66354aa5f84b8cc26f1764)

**Review by ihortok — APPROVED** [review]

**Pull request merged**